src/main/java/edu/cornell/kfs/module/cam/rest/resource/CuCapAssetInventoryApiResource.java

```diff
@@ -184,18 +187,44 @@ public Response updateAsset(@PathParam(CuCamsConstants.CapAssetApi.ASSET_TAG_PAR
             return ex instanceof BadRequestException ? respondBadRequest() : respondInternalServerError(ex);
         }
     }
+    
+    @GET
+    @Path("/campusCodes")
+    public Response getCampusCodes(@Context HttpHeaders headers) {
+        try {
+            List<Map<String, String>> campusCodes = new ArrayList<>();
+            
+            Map<String, Object> fieldValues = new HashMap<>();
+            fieldValues.put(CuCamsConstants.CapAssetApi.ACTIVE, true);
+            Collection<org.kuali.kfs.sys.businessobject.Campus> campuses = 
+                SpringContext.getBean(BusinessObjectService.class).findMatching(
```

Comment:
Instead of embedding the `SpringContext.getBean()` call here, you may want to separate it out into a separate method, like what this class does with its other service references.